.run/Runelite.run.xml

```diff
@@ -1,11 +1,11 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Runelite" type="Application" factoryName="Application" singleton="false">
-    <option name="ALTERNATIVE_JRE_PATH" value="11" />
+    <option name="ALTERNATIVE_JRE_PATH" value="jbr-21" />
     <option name="ALTERNATIVE_JRE_PATH_ENABLED" value="true" />
     <option name="MAIN_CLASS_NAME" value="net.runelite.client.RuneLite" />
     <module name="client" />
-    <option name="PROGRAM_PARAMETERS" value="-developer-mode -clean-jagex-launcher" />
-    <option name="VM_PARAMETERS" value="-ea" />
+    <option name="PROGRAM_PARAMETERS" value="-developer-mode" />
+    <option name="VM_PARAMETERS" value="-ea --add-opens=java.desktop/com.apple.eawt=ALL-UNNAMED --add-opens=java.desktop/sun.awt=ALL-UNNAMED" />
```

Comment:
please leave this config out of the PR

Reply:
Reverted change to Runelite.run.xml to retain default settings.